packages/dialog/package.json

```diff
@@ -48,6 +48,7 @@
   },
   "devDependencies": {
     "@esm-bundle/chai": "^4.3.4",
+    "@vaadin/a11y-base": "24.1.0-alpha9",
```

Comment:
Do not forget to update this to `24.1.0-alpha10`

Reply:
Done.